docs/metasploit-framework.wiki/Hashes-and-Password-Cracking.md

```diff
@@ -1,6 +1,6 @@
 # Intro
 
-This article will discuss the various libraries, dependencies, and functionality built in to metasploit for dealing with password hashes, and cracking them.  In general, this will not cover storing credentials in the database, which can be read about [here](https://github.com/rapid7/metasploit-framework/wiki/Creating-Metasploit-Framework-LoginScanners#the-scan-block).  Metasploit currently support cracking passwords with [John the Ripper](https://github.com/rapid7/metasploit-framework/tree/master/modules/auxiliary/analyze) and  [hashcat](https://github.com/rapid7/metasploit-framework/pull/11695).
+This article will discuss the various libraries, dependencies, and functionality built in to metasploit for dealing with password hashes, and cracking them.  In general, this will not cover storing credentials in the database, which can be read about [here](https://docs.metasploit.com/docs/development/developing-modules/guides/scanners/creating-metasploit-framework-loginscanners.html#the-scan-block).  Metasploit currently support cracking passwords with [John the Ripper](https://github.com/rapid7/metasploit-framework/tree/master/modules/auxiliary/analyze) and  [hashcat](https://github.com/rapid7/metasploit-framework/pull/11695).
```

Comment:
Long term we shouldn't have direct links in the wiki files; they should use relative links so that we can change the base URL in the future

Reply:
I was thinking of that as well, I agree changing these to relative links would be ideal.

Reply:
Not tested - but I believe we'll want to change this to use the syntax `[[here|Creating-Metasploit-Framework-LoginScanners]]`

Otherwise as these direct links that you're adding won't get validated as part of the `navigation.rb` entries

Reply:
From our Slack discussion:

It only works with the Github wiki flavor markdown syntax. I don't believe the anchor links are supported though (i.e. foo.md#section) 🤔 
https://github.com/rapid7/metasploit-framework/blob/953621860af4b7a83426e67ede67b479865d9b78/docs/build.rb#L206-L210

I'd prefer using the `[[Custom name|./relative-path.md]]` syntax for now, since it'll be easier to autocomplete paths in IDEs, and easier to migrate to the newer `[text](link)` syntax in the future

Reply:
Should be fixed now with https://github.com/rapid7/metasploit-framework/pull/17426/commits/2a43d96e82d801877625ab45a4ca12fc3c02bd83